Framework/LiveData/inc/MantidLiveData/Kafka/IKafkaStreamDecoder.h

```diff
@@ -57,10 +82,12 @@ class DLLExport IKafkaStreamDecoder {
 
   ///@name Callbacks
   ///@{
-  virtual void registerIterationEndCb(CallbackFn cb) {
-    m_cbIterationEnd = std::move(cb);
+  virtual void registerIterationEndCb(Callback::FnType cb) {
```

Comment:
Using std::move on a member function is a code smell. Can this be passed by const ref?